build/config/tizen/config.gni

```diff
@@ -14,5 +14,6 @@
 
 declare_args() {
   # Location of the Tizen SDK.
```

Comment:
This comment line is for `tizen_sdk_root`. Modify it so it will cover these both variables.